marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/graph/nodeExpand.sjs

```diff
@@ -74,6 +90,10 @@ limit = limit || 100;
 let queryObj = JSON.parse(nodeInfo);
 const nodeToExpand = queryObj.parentIRI;
 
+let isConcept = false;
+if(queryObj.isConcept != null && queryObj.isConcept === "true"){
```

Comment:
This might be more of a question for the front-end, but is there a reason we're using the string "true" instead of the boolean value?

Reply:
Good catch @ryanjdew ! I was talking with front team, and this should be a boolean as you mentioned. I already changed it, thanks!